Lib/ufo2ft/makeotfParts.py

```diff
@@ -131,7 +134,7 @@ def setupAnchorPairs(self):
         for glyph in self.font:
             for anchor in glyph.anchors:
                 if anchor.name is None:
-                    print("warning: unnamed anchor discarded in", glyph.name)
+                    logger.warn("Unnamed anchor discarded in %s" % glyph.name)
```

Comment:
Nit: if you pass positional arguments instead of using '%' straight away, that expensive string formatting is evaluated only when/if the log message is actually emitted.

```diff
-logger.warn("Unnamed anchor discarded in %s" % glyph.name)
+logger.warn("Unnamed anchor discarded in %s", glyph.name)
```